### ISSUE TITLE
Fails with "image: unknown format"

### ISSUE THREAD

**Issue by robinovitch61** [robinovitch61/webtoon-dl#5]
From https://www.reddit.com/r/webtoons/comments/18nsej2/comment/ksi25ro/?utm_source=share&utm_medium=web2x&context=3

I've been getting this (https://ibb.co/bJL9Sf5) error message whenever I try to download a webtoon. The first 70 chapters downloaded fine, but it throws the same error after it gets partially through 70-80. I've tried adjusting the starting chapter to skip a few chapters, but it recieves the same error. Here's what I put in after changing it around: (https://ibb.co/b1HrJZr).

What I input:

```
webtoon-dl --min-ep=75 --max-ep=180 "https://www.webtoons.com/en/comedy/cursed-princess-club/list?title_no=1537"
```

Please help!

Thank you so much for making this tool! I really love it.

**Comment by mojamm11** [comment]
i got the same issue when i used it to download a webtoon in the same numbering range of episodes

**Comment by robinovitch61** [comment]
Fixed in v0.0.8 - there were gifs in this comic, I made it so they are skipped:
```
saving episodes 78 through 79: added page 89/247
saving episodes 78 through 79: added page 90/247
WARNING: skipping gif https://webtoon-phinf.pstatic.net/20200803_172/1596413306033xHnBC_GIF/15964133059451537785.gif?type=ani
saving episodes 78 through 79: added page 92/247
saving episodes 78 through 79: added page 93/247
```
